src/plugins/discover/public/build_services.ts

```diff
@@ -111,7 +110,7 @@ export interface DiscoverServices {
   contentClient: ContentClient;
 }
 
-export const buildServices = memoize(function (
+export const buildServices = function (
```

Comment:
What caused this change to be needed?

Reply:
This is was also an old artefact. Good catch.. I have now reverted it.